README.md

```diff
@@ -12,19 +12,28 @@ This project is currently under development and can have breaking API changes.
 
 ---
 
-## Generating an Xcode Project
+## Requirements
 
-To see this project locally in Xcode you can run the following command on the root of this project:
-`swift package generate-xcodeproj`
+- Swift 5.1
```

Comment:
Could be also to list the version of Xcode you need to have.

Reply:
It's not trully _required_ to use Xcode. You could develop with something like VS Code, but it would still require Swift 5.1., so I'd prefer to keep this constraint to the actual requirement of the codebase.